packages/bpk-component-heading/src/BpkHeading.js

```diff
@@ -16,27 +16,38 @@
  * limitations under the License.
  */
 
+/* @flow strict */
+
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { type Node } from 'react';
 import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './BpkHeading.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkHeading = props => {
-  const classNames = [getClassName(`bpk-heading-${props.level}`)];
+type Props = {
+  children: Node,
+  level: string,
```

Comment:
Can we make this more strict to reflect the prop type?

i.e. `'h1' | 'h2' ... etc`

Reply:
Done